questdb-rs/src/ingress/mod.rs

```diff
@@ -24,8 +24,8 @@
 
 #![doc = include_str!("mod.md")]
 
+pub use self::ndarr::*;
```

Comment:
The `*` risks leaking too many types.
Instead just pick the ones you explicitly are sure you want in the public API.